cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -83,6 +83,8 @@ spec:
       #public: public-conf --> NetworkAttachmentDefinition object name in Multus
       #cluster: cluster-conf --> NetworkAttachmentDefinition object name in Multus
   # enable the crash collector for ceph daemon crash collection
+    # Provide internet protocol version. IPv6, IPv4 or empty string are valid options. Empty string would mean IPv4
```

Comment:
You need to add this before `# enable the crash collector for ceph daemon crash collection`

Reply:
fixed.